api/incidents/incidentsRouter.js

```diff
@@ -455,6 +458,23 @@ router.get('/download', async (req, res) => {
       incidents = filterDataByDate(incidents, range);
     }
 
```

Comment:
spacing here is different than lines 465 and 466. please pick one convention and go with that.